src/gh.ts

```diff
@@ -1268,16 +1410,79 @@ const completionSpec: Fig.Spec = {
               description: "Show only public repositories",
             },
             { name: "--source", description: "Show only non-forks" },
+
+            {
+              name: "-q, --jq",
+              description: "Filter JSON output using a jq expression",
+            },
+            {
+              name: "--json",
+              description: "Output JSON with the specified fields",
+            },
+            {
+              name: "-t, --template",
+              description: "Format JSON output using a Go template",
+            },
+          ],
+        },
+        {
+          name: "rename",
+          description: `Rename a GitHub repository.
+          By default, this renames the current repository; otherwise renames the specified repository.`,
+          args: {
+            name: "new-name",
+            isOptional: true,
+          },
+          options: [ghOptions.help, ghOptions.confirm, ghOptions.all],
+        },
+        {
+          name: "sync",
+          description: `Sync destination repository from source repository. Syncing uses the main branch
+          of the source repository to update the matching branch on the destination
+          repository so they are equal. A fast forward update will be used execept when the
+          '--force' flag is specified, then the two branches will
+          by synced using a hard reset.
+          
+          Without an argument, the local repository is selected as the destination repository.
+          
+          The source repository is the parent of the destination repository by default.
+          This can be overridden with the '--source' flag.`,
+          args: {
+            name: "destination-repository",
+            isOptional: true,
+          },
+          options: [
+            ghOptions.help,
+
+            {
+              name: ["-b", "--branch string"],
+              description: "Branch to sync (default: main branch)",
+            },
+            {
+              name: "--force",
+              description:
+                "Hard reset the branch of the destination repository to match the source repository",
+            },
+            {
+              name: ["-s", "--source string"],
+              description: "Source repository",
+            },
           ],
         },
         {
           name: "view",
-          description: "View a repository",
+          description: `Display the description and the README of a GitHub repository.
+          With no argument, the repository for the current directory is displayed.
+          With '--web', open the repository in a web browser instead.
+          With '--branch', view a specific branch of the repository.
+          For more information about output formatting flags, see 'gh help formatting'.
+          `,
```

Comment:
We need to use this weird indentation to avoid extra spaces:
```suggestion
With no argument, the repository for the current directory is displayed.
With '--web', open the repository in a web browser instead.
With '--branch', view a specific branch of the repository.
For more information about output formatting flags, see 'gh help formatting'`,
```